README.md

```diff
@@ -111,6 +111,18 @@ The `gitbook` command by default will create a folder called `exports` that cont
 
 Running the `docx` command will by default create a file named `export.docx` in the local directory.
 
+## Gap Analysis
+
+1. Once you collect the dependencies via the `get` command, run:
+
+    ```bash
+    $ compliance-masonry diff
+    Number of missing controls: 3
+    AT-4
+    AU-12
+    ...
+    ```
```

Comment:
I agree. There was little problem that I came across and how I'm going to still fix this inefficiency in the next commit.

I'm anticipating that in the future we will want to do more detailed gap analysis (e.g. show gap analysis for which controls are marked as "partially" or "not" implemented. [example](https://github.com/18F/cg-compliance/blob/a8c118a91ce61017c913948d4ebc063a0a556158/AC_Policy/component.yaml#L13)) In order to do that, I'm going to need the full object and not just unique strings. Which is all good but the type of object in `masterControlList` is of type `models.Control` and the ones given to us by the ones actually documented, `actualSatisfiedControls`, are of `models.Satisfies`. Which won't work out the box with set algebra.

New solution
```go
type Inventory struct {
	*models.OpenControl
	masterControlList       map[string]models.Control
	actualSatisfiedControls map[string]models.Satisfies
	MissingControlList      map[string]models.Control
}
```
The keys for `masterControlList` and `actualSatisfiedControls` are those unique strings with the `standard@control` but each map will contains their respective object as the value. I can do quick look ups and comparisons without the double for-loop and in the future we can pull out more details from each of the objects.